server/WebAPI/DTO/Enums/AccommodationSortBy.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Entities.Models.Enums
+{
+  public enum AccommodationSortBy
+  {
+    ByReservationDate, ByCostPerNight, ByName
+  }
```

Comment:
public enum AccommodationSortBy
  {
    ByReservationDate,
    ByCostPerNight,
    ByName
  }

it reads better this way